### PR TITLE
trying auto deploy to production

### PR DESCRIPTION
**Pull request by paulijones**

### CODE REVIEW

**Comment by paulijones** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **paulijones**

If they are not already assigned, you can assign the PR to them by writing `/assign @paulijones` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=cheddartv%2Fgolang-jx-quickstart).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/cheddartv/golang-jx-quickstart/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by paulijones** [comment]
:star: PR built and available in a preview environment **cheddartv-golang-jx-quickstart-pr-2** [here](http://golang-jx-quickstart.jx-cheddartv-golang-jx-quickstart-pr-2.jx.curds.io)

File: main.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func handler(w http.ResponseWriter, r *http.Request) {
-	title := "Jenkins X golang http example is fun!"
+	title := "Jenkins X golang http example is fun, and should go to production on merge!"
```
Comment (rosscooperman) [inline main.go]: I don't think this comma is necessary

**Comment by paulijones** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/cheddartv/golang-jx-quickstart/pull/2#pullrequestreview-271589929" title="Approved">rosscooperman</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=cheddartv%2Fgolang-jx-quickstart).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/cheddartv/golang-jx-quickstart/blob/master/OWNERS)~~ [rosscooperman]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**